packages/calypso-products/src/constants/types.ts

```diff
@@ -12,6 +12,7 @@ export const TYPE_ALL = 'TYPE_ALL';
 export const TYPE_P2_PLUS = 'TYPE_P2_PLUS';
 export const TYPE_FLEXIBLE = 'TYPE_FLEXIBLE';
 export const TYPE_PRO = 'TYPE_PRO';
+export const TYPE_STARTER = 'TYPE_STARTER'; // @todo find use
```

Comment:
Weird we have these types and also the other product constants in [wpcom.ts](https://github.com/Automattic/wp-calypso/blob/update/plans-overhaul-starter/packages/calypso-products/src/constants/wpcom.ts). I think i can tell what they refer to, but still, having both seems redundant. Also, unless mistaken, we condition on both at places for same purpose e.g. in [plans-comparison-action.tsx](https://github.com/Automattic/wp-calypso/blob/update/plans-overhaul-starter/client/my-sites/plans-comparison/plans-comparison-action.tsx)?

Reply:
Moved to issue https://github.com/Automattic/wp-calypso/issues/63497